### PR TITLE
fix(test):modify tests to support local runs

### PR DESCRIPTION
**Pull request by rgarg1**
The tests now can run on WIT running on localhost

### CODE REVIEW

File: EE_API_automation/pytest/src/test_planner.py [hunk 1]
```diff
@@ -14,6 +15,10 @@ def test_setup_planner(self, sut, offline_token, userid):
             launch_detail.base_url[launch_detail.base_wit] = r"https://api.openshift.io"
             print "SUT (WIT Target) not provided!!! Using default production SUT = ", launch_detail.base_url[launch_detail.base_wit]
         else:
+            # Identify if its a local run and set the local_run variable to True
+            if "localhost" in sut or "0.0.0.0" in sut:
```
Comment (jarifibrahim) [inline EE_API_automation/pytest/src/test_planner.py]: Add `127.0.0.1` also

File: EE_API_automation/pytest/src/test_planner.py [hunk 2]
```diff
@@ -24,15 +29,24 @@ def test_setup_planner(self, sut, offline_token, userid):
             launch_detail.userid_primary = userid
             print "USERID set to = ", launch_detail.userid_primary
 
-        if offline_token is None:
-            pytest.exit("OFFLINE_TOKEN not provided!!! Terminating the run!!!!!!!!!!!")
+        if offline_token in ["", "0", False, 0, None, "None"]:
+            if local_run:
+                try:
+                    launch_detail.token_userid_primary = launch_detail.get_local_token()
+                    if launch_detail.token_userid_primary is not None:
```
Comment (jarifibrahim) [inline EE_API_automation/pytest/src/test_planner.py]: `if launch_detail.token_userid_primary` should be enough

File: EE_API_automation/pytest/src/test_planner.py [hunk 3]
```diff
@@ -24,15 +29,24 @@ def test_setup_planner(self, sut, offline_token, userid):
...
         else:
             launch_detail.offref_token_userid_primary = offline_token
-            launch_detail.token_userid_primary = launch_detail.get_access_token_from_refresh()
-            if launch_detail.token_userid_primary is None:
-                pytest.exit("ACCESS_TOKEN cannot be generated!!! Terminating the run!!!!!!!!!!!")
-            else:
-                print "ACCESS_TOKEN set to = A secret in Jenkins ;)"
+            try:
+                launch_detail.token_userid_primary = launch_detail.get_access_token_from_refresh()
+                if launch_detail.token_userid_primary is not None:
```
Comment (jarifibrahim) [inline EE_API_automation/pytest/src/test_planner.py]: `if launch_detail.token_userid_primary ` should be enough

**Comment by rgarg1** [comment]
@jarifibrahim Addressed your comments and some more...

**Pull request merged**